armi/reactor/tests/test_parameters.py

```diff
@@ -509,228 +523,80 @@ class MockSyncPC(parameters.ParameterCollection):
-                        getattr(self, methodName)()
-                    except Exception:
-                        self.write("failed, big time")
-                        traceback.print_exc(file=self.l)
-                        self.write("*** printed exception")
-                        try:
-                            self.tearDown()
-                        except:  # noqa: bare-except
-                            pass
-
-            self.l.write("done.")
-
-    def write(self, msg):
-        self.l.write("{}\n".format(msg))
-        self.l.flush()
-
-    def assertRaises(self, exceptionType):
-        class ExceptionCatcher:
-            def __enter__(self):
-                pass
-
-            def __exit__(self, exc_type, exc_value, traceback):
-                if exc_type is exceptionType:
-                    return True
-                raise AssertionError(
-                    "Expected {}, but got {}".format(exceptionType, exc_type)
-                )
-
-        return ExceptionCatcher()
+    @unittest.skipIf(context.MPI_SIZE <= 1 or MPI_EXE is None, "Parallel test only")
+    def test_noConflicts(self):
+        """Make sure sync works across processes.
 
-    def assertEqual(self, expected, actual):
-        if expected != actual:
-            raise AssertionError(
-                "(expected) {} != {} (actual)".format(expected, actual)
-            )
-
-    def assertNotEqual(self, expected, actual):
-        if expected == actual:
-            raise AssertionError(
-                "(expected) {} == {} (actual)".format(expected, actual)
-            )
+        .. test:: Synchronize a reactor's state across processes.
+            :id: T_ARMI_CMP_MPI0
+            :tests: R_ARMI_CMP_MPI
+        """
+        syncCount = self.r.syncMpiState()
 
-    def mpitest_noConflicts(self):
         for ci, comp in enumerate(self.comps):
             if ci % context.MPI_SIZE == context.MPI_RANK:
                 comp.p.param1 = (context.MPI_RANK + 1) * 30.0
             else:
                 self.assertNotEqual((context.MPI_RANK + 1) * 30.0, comp.p.param1)
 
-        self.assertEqual(len(self.comps), self.r.syncMpiState())
+        syncCount = self.r.syncMpiState()
+        self.assertEqual(len(self.comps), syncCount)
 
         for ci, comp in enumerate(self.comps):
             self.assertEqual((ci % context.MPI_SIZE + 1) * 30.0, comp.p.param1)
 
-    def mpitest_noConflicts_setByString(self):
-        """Make sure params set by string also work with sync."""
-        for ci, comp in enumerate(self.comps):
-            if ci % context.MPI_SIZE == context.MPI_RANK:
-                comp.p.param2 = (context.MPI_RANK + 1) * 30.0
-            else:
-                self.assertNotEqual((context.MPI_RANK + 1) * 30.0, comp.p.param2)
+    @unittest.skipIf(context.MPI_SIZE <= 1 or MPI_EXE is None, "Parallel test only")
+    def test_withConflicts(self):
+        """Test conflicts arrise correctly if we force a conflict.
```

Comment:
```suggestion
        """Test conflicts arise correctly if we force a conflict.
```

Reply:
Not sure why spellcheck is failing me. But good catch. 

Fixed.